src/Uno.UWP/Devices/Lights/Lamp.unsupported.cs

```diff
@@ -0,0 +1,14 @@
+﻿#if !__ANDROID__ && !__IOS__
+
+namespace Windows.Devices.Lights
+{
+	public partial class Lamp
+	{
+		/// <summary>
+		/// API not supported, always returns null.
+		/// </summary>
+		/// <returns>Null.</returns>
+		public static IAsyncOperation<Lamp> GetDefaultAsync() => Task.FromResult<Lamp>(null).AsAsyncOperation();
```

Comment:
Not implemented attribute?

Reply:
In this case I think it is okay - the sensor itself is not available on the target platforms (WPF, GTK, macOS), so returning `null` is valid - when the device doesn't have a `Lamp`, the API does return `null`. The advantage of this as compared to having the method throw is that the developer can use the code without any `#if` - comfortable that the code will never enter the branch after `if (lamp != null)`